pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/RuntimeUtils.java

```diff
@@ -320,6 +320,11 @@ public static List<String> getCmd(InstanceConfig instanceConfig,
 
             args.add("-Dio.netty.tryReflectionSetAccessible=true");
 
+            // Needed for netty.DnsResolverUtil on JDK9+
+            if (!System.getProperty("java.version").startsWith("1.")) {
```

Comment:
what about using commons-lang3 utility 
`SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_9)` ?

Reply:
we should have a consistent approach across the code base, do you see other cases in Java code where we detect the Java version ?

Reply:
No. That's the only place I found